src/data/ScopeClosures/ch1.js

```diff
@@ -146,7 +146,7 @@ const Ch1Questions = [
     moreInfoUrl:
       'https://github.com/getify/You-Dont-Know-JS/blob/master/scope%20%26%20closures/ch1.md#compiler-speak',
     explanation:
-      'The reference to `a` in `console.log( a );` is an RHS reference, because nothing is being assigned to a here. By contrast, in `a = 2;` the reference to `a` is an LHS reference, because we don&#39;t actually care what the current value is. We simply want to find the variable as `a` target for the `= 2` assignment operation.',
```

Comment:
I think it would be best to keep "a" in this case, but make sure it is formatted as `a`

Reply:
Didn't realize that the "a" in that case was referencing a variable, thought it was just a typo. Good catch and thanks for the quick reply. Formatted it in my next commit.